requirements-python.txt

```diff
@@ -20,8 +20,6 @@ PTable==0.9.2
 pycryptodome==3.8.1
 pyzmq==18.0.1
 tenacity==5.0.4
-# this package isn't in pypi
```

Comment:
why are you removing this, please leave this package in

Reply:
if you want to remove this clone from private branch, please code all the code into our repo.
why rewriting it ?

Reply:
you can opne a trello task to handle this repo, i.e. by uploading to pypi
or by copying it into out code base